modules/gocd/conf/cruise-config.xml

```diff
@@ -122,7 +117,25 @@
               <exec command="/bin/bash" workingdir="#{SOURCE_DIR}">
                 <arg>-c</arg>
                 <arg>newTag=#{APP_DOCKER_IMAGE}:#{APP_IMAGE_BUILD_VERSION};\
-                     sudo docker push ${newTag}</arg>
+                     sudo docker push ${newTag};
+                     echo "Delete Empty Docker Images ...";
```

Comment:
when we tag new docker images as "latest" the older ones have empty tag and name. Refers to that.